x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/index.test.tsx

```diff
@@ -0,0 +1,756 @@
+
+jest.mock('../../../../lib/kibana');
+jest.mock('../../../../containers/case/configure/use_connectors');
+jest.mock('../../../../containers/case/configure/use_configure');
+jest.mock('../../../../components/navigation/use_get_url_search');
+
+const useKibanaMock = useKibana as jest.Mock;
+const useConnectorsMock = useConnectors as jest.Mock;
+const useCaseConfigureMock = useCaseConfigure as jest.Mock;
+const useGetUrlSearchMock = useGetUrlSearch as jest.Mock;
+
+import { ConfigureCases } from './';
+import { TestProviders } from '../../../../mock';
+import { createUseKibanaMock } from '../../../../mock/kibana_react';
+import { actionTypeRegistryMock } from '../../../../../../../../plugins/triggers_actions_ui/public/application/action_type_registry.mock';
+import { Connectors } from './connectors';
+import { ClosureOptions } from './closure_options';
+import { Mapping } from './mapping';
+import {
+  ActionsConnectorsContextProvider,
+  ConnectorAddFlyout,
+  ConnectorEditFlyout,
+} from '../../../../../../../../plugins/triggers_actions_ui/public';
+import { EuiBottomBar } from '@elastic/eui';
+
+const useCaseConfigureResponse: ReturnUseCaseConfigure = {
+  loading: false,
+  persistLoading: false,
+  refetchCaseConfigure: jest.fn(),
+  persistCaseConfigure: jest.fn(),
+};
+
+const useConnectorsResponse: ReturnConnectors = {
+  loading: false,
+  connectors,
+  refetchConnectors: jest.fn(),
+};
+
+const kibanaMockImplementationArgs = {
+  services: {
+    ...createUseKibanaMock()().services,
+    triggers_actions_ui: { actionTypeRegistry: actionTypeRegistryMock.create() },
+  },
+};
+
+describe('ConfigureCases', () => {
+  let wrapper: ReactWrapper;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useCaseConfigureMock.mockImplementation(
+      ({ setConnector, setClosureType, setCurrentConfiguration }) => {
+        useEffect(() => setConnector('123'), []);
+        useEffect(() => setClosureType('close-by-user'), []);
+        useEffect(
+          () =>
+            setCurrentConfiguration({
+              connectorId: '123',
+              closureType: 'close-by-user',
+            }),
+          []
+        );
+        return useCaseConfigureResponse;
+      }
+    );
+    useConnectorsMock.mockImplementation(() => useConnectorsResponse);
+    useKibanaMock.mockImplementation(() => kibanaMockImplementationArgs);
+    useGetUrlSearchMock.mockImplementation(() => searchURL);
+
+    wrapper = mount(<ConfigureCases userCanCrud />, { wrappingComponent: TestProviders });
+  });
+
+  test('it renders correctly', () => {
+    expect(wrapper.find(Connectors).exists()).toBeTruthy();
+    expect(wrapper.find(ClosureOptions).exists()).toBeTruthy();
+    expect(wrapper.find(Mapping).exists()).toBeTruthy();
+    expect(wrapper.find(ActionsConnectorsContextProvider).exists()).toBeTruthy();
+    expect(wrapper.find(ConnectorAddFlyout).exists()).toBeTruthy();
+    expect(wrapper.find(ConnectorEditFlyout).exists()).toBeTruthy();
```

Comment:
Seems that this is the case after data is fetched, I would like to have that in the description